css-overflow-4/Overview.bs

```diff
@@ -1140,6 +1170,55 @@ Fragmentation of Overflow: the 'continue' property</h3>
 	as the box would be a [=flex container=] rather than a [=block container=],
 	and thus the 'continue' property would not apply.
 
+<h4 id=line-clamp-containers>Line-clamp containers</h4>
+
+	A <dfn>line-clamp container</dfn> is an [=independent formatting context|independent=] [=block formatting context=]
+	that additionally follows the rules in this section.
+	All line-clamp containers contain a <dfn>clamp point</dfn>,
+	which is one of the following positions inside it:
+
+	- A point immediately after a [=line box=]
+		in the line-clamp container's [=block formatting context=].
+	- The end of the line-clamp container,
+		if there are any boxes after the last [=inline formatting context=]
+		in the line-clamp container's [=block formatting context=].
+
+	If the [=line-clamp container=]'s [=block formatting context root=]
+	has a [=computed value=] of 'max-lines' other than ''max-lines/none'',
+	then that property will determine the [=clamp point=].
+	Otherwise, the clamp point will be set to the last possible clamp point
+	such that, for it and all previous possible clamp points,
+	the line-clamp container's [=automatic block size=]
+	(as determined below)
+	is not greater than the [=block size=] the box would have
+	if its automatic block size were infinite;
+	or if that is not the case for any possible clamp points,
+	to the first possible clamp point in the block formatting context.
```

Comment:
This could use an example of what the "Otherwise" behavior is trying to guard against.

Reply:
It's not guarding against anything. In this PR, currently, `line-clamp: 3` will set `max-lines: 3`, and that will clamp based on the number of lines; while `line-clamp: auto` will set `max-lines: none`, which will clamp based on the height. This height-based clamping is what the entire "otherwise" clause describes.

I will definitely have to do some rewriting and clarification, though, because this is definitely not obvious from just skimming the spec.